openlibrary/catalog/marc/tests/test_data/xml_expect/00schlgoog.json

```
@@ -16,20 +16,23 @@
   "publish_country": "fr",
   "authors": [
     {
-      "name": "Yehudai ben Na\u1e25man gaon",
+      "name": "Yehudai ben Naḥman gaon",
       "title": "gaon",
-      "personal_name": "Yehudai ben Na\u1e25man",
+      "personal_name": "Yehudai ben Naḥman",
       "role": "supposed author",
       "date": "8th century",
       "entity_type": "person"
+    },
+    {
+      "name": "Schlosberg, Leon",
+      "date": "d. 1899",
+      "entity_type": "person",
+      "role": "ed"
```

```suggestion
      "role": "ed."
```
Keep trailing period